logger.c

```diff
@@ -206,7 +206,7 @@ static int _logger_parse_ise(logentry *e, char *scratch) {
     const char * const status_map[] = {
         "not_stored", "stored", "exists", "not_found", "too_large", "no_memory" };
     const char * const cmd_map[] = {
-        "null", "add", "set", "replace", "append", "prepend", "cas" };
+        "null", "add", "set", "replace", "append", "prepend", "cas", "append", "prepend" };
 
     if (le->cmd <= 6)
         cmd = cmd_map[le->cmd];
```

Comment:
you would need to change this to length of the `cmd_map`, it's not working currently:
```
echo watch fetchers mutations | nc localhost 11211
OK
ts=1678384796.15470 gid=2 type=item_get key=append-test status=not_found clsid=0 cfd=25 size=0
ts=1678384796.15477 gid=3 type=item_store key=append-test status=stored cmd=na ttl=98 clsid=1 cfd=25 size=5
ts=1678384824.945079 gid=4 type=item_get key=append-test status=found clsid=1 cfd=25 size=5
ts=1678384824.945082 gid=5 type=item_store key=append-test status=stored cmd=na ttl=69 clsid=1 cfd=25 size=10
ts=1678384830.459650 gid=6 type=item_get key=append-test status=found clsid=1 cfd=25 size=10
ts=1678384838.767296 gid=7 type=item_get key=append-test status=found clsid=1 cfd=25 size=10
ts=1678384850.559499 gid=8 type=item_get key=append-test status=found clsid=1 cfd=25 size=10
ts=1678384850.559504 gid=9 type=item_store key=append-test status=stored cmd=na ttl=44 clsid=1 cfd=25 size=15
ts=1678384856.493884 gid=10 type=item_get key=append-test status=found clsid=1 cfd=25 size=15
```

Reply:
the `cmd=na` should be `cmd=append`

Reply:
Whoops, fixed thanks.